addon/components/power-calendar/days.js

```diff
@@ -78,7 +78,7 @@ export default Component.extend({
         daysOfWeek = daysOfWeek.filter((d) => d.isCurrentMonth);
       }
       weeks.push({
-        id: days[0].moment.format('YYYY-w'),
+        id: days[i].moment.format('YYYY-w'),
```

Comment:
`days` contains all visible days, so it uses the week number of the first visible day in every week iteration.
`daysOfWeek[0]` would also work.